packages/webdriverio/tests/findStrategy.test.ts

```diff
@@ -113,25 +113,25 @@ describe('selector strategies helper', () => {
     it('should find an element by tag name + class + content', () => {
         const element = findStrategy('div.some-class=some random text with "§$%&/()div=or others')
         expect(element.using).toBe('xpath')
-        expect(element.value).toBe('.//div[contains(@class, "some-class") and normalize-space(text()) = "some random text with "§$%&/()div=or others"] | .//div[contains(@class, "some-class") and not(.//div[contains(@class, "some-class") and normalize-space(text()) = "some random text with "§$%&/()div=or others"]) and normalize-space() = "some random text with "§$%&/()div=or others"]')
+        expect(element.value).toBe('.//div[contains(concat(" ",@class," "), " some-class ") and normalize-space(text()) = "some random text with "§$%&/()div=or others"] | .//div[contains(concat(" ",@class," "), " some-class ") and not(.//div[contains(concat(" ",@class," "), " some-class ") and normalize-space(text()) = "some random text with "§$%&/()div=or others"]) and normalize-space() = "some random text with "§$%&/()div=or others"]')
```

Comment:
Do you know why:

```
concat(" ",@class," "), " some-class ")
```
differs from
```
contains(@class, "some-class")
```
?

Reply:
Yeah, contains "bar" matches "foo-bar" whereas instead we want to look for "bar".

Now you can also use @class="bar" but this has the downside that when you have an element with a class "foo bar baz" it Will not match.

The solution I applied matches a class in a classlist without matching too much (classes that only match partially)

Reply:
I think I updated the PR description after your feedback was placed. I think the description shows a good example as well

Reply:
That makes sense, thanks! Can you confirm that `concat(" ",@class," "), " foo ")` matches all of the following elements:

- `<div class="foo bar baz"></div>`
- `<div class="bar foo baz"></div>`
- `<div class="bar baz foo"></div>`

I am asking because the selector suggests that there has to be a space left and right from the class name.

Reply:
Better yet, I'll add a testcase for it ;)